.circleci/config.yml

```diff
@@ -79,7 +85,7 @@ jobs:
       - restore_cache:
           name: Restore Yarn Package Cache
           keys:
-            - yarn-{{ parameters.e }}-{{ checksum "yarn.lock" }}
+            - yarn-{{ $EXECUTOR_NAME }}-{{ checksum "yarn.lock" }}
```

Comment:
AFAIK you can't use arbitrary environment variables in cache keys. You can use `{{ .Environment.variableName }}` but only for environment variables that CircleCI defines.

Reply:
As a workaround you could `echo "$EXECUTOR_NAME" > /tmp/.executor_name` and then do `{{ checksum "/tmp/.executor_name" }}`.